calicoctl/calico_ctl/node.py

```diff
@@ -881,123 +783,20 @@ def handle_sigterm(sig, frame):
         docker_client.stop(container)
         sys.exit(0)
     signal.signal(signal.SIGTERM, handle_sigterm)
+    stop_container_on_exit = True
 
     output = docker_client.attach(container, stream=True)
     try:
         for raw_data in output:
             sys.stdout.write(raw_data)
+            if "Calico node started successfully" in raw_data:
+                stop_container_on_exit = False
+                break
     except KeyboardInterrupt:
         # mainline.  someone press Ctrl-C.
```

Comment:
You didn't do this comment but it is yucky... What about: "**M**ainline.  **S**omeone press**ed** Ctrl-C"